config/config.go

```diff
@@ -1050,6 +1050,7 @@ func SetServerDefaults(v *viper.Viper) error {
 		v.SetDefault(param.Director_GeoIPLocation.GetName(), "/var/cache/pelican/maxmind/GeoLite2-City.mmdb")
 		v.SetDefault(param.Registry_DbLocation.GetName(), "/var/lib/pelican/registry.sqlite")
 		v.SetDefault(param.Director_DbLocation.GetName(), "/var/lib/pelican/director.sqlite")
+		v.SetDefault(param.Cache_DbLocation.GetName(), "/var/lib/pelican/cache.sqlite")
```

Comment:
I think this is still missing a default for non-root runners. Can you check around line 1064 to see how we set that up for other databases and add one?